docs/index.md

```diff
@@ -70,12 +70,13 @@ Note that LSI will work without these libraries, but as soon as they are install
 
 * Install [LAPACKE](https://www.netlib.org/lapack/lapacke.html)
   * Ubuntu: `apt-get install liblapacke-dev`
-  * macOS: (Help wanted to verify installation steps) https://stackoverflow.com/questions/38114201/installing-lapack-and-blas-libraries-for-c-on-mac-os
+  * macOS: `brew install lapack`
 * Install [OpenBLAS](https://www.openblas.net/)
   * Ubuntu: `apt-get install libopenblas-dev`
-  * macOS: (Help wanted to verify installation steps) https://stackoverflow.com/questions/38114201/installing-lapack-and-blas-libraries-for-c-on-mac-os
-* Install the [Numo::NArray](https://ruby-numo.github.io/narray/) and [Numo::Linalg](https://ruby-numo.github.io/linalg/) gems
-  * `gem install numo-narray numo-linalg`
+  * macOS: `brew install openblas`
+* Install the [Numo::NArray](https://ruby-numo.github.io/narray/) and [Numo::Linalg](https://ruby-numo.github.io/linalg/) gems. If you're using Bundler, add `numo-narray` and `numo-linalg` to your Gemfile.
+  * Ubuntu: `gem install numo-narray numo-linalg`
+  * macOS: `gem install numo-narray`, `gem install numo-linalg -- --with-openblas-dir=/opt/homebrew/opt/openblas --with-lapack-lib=/opt/homebrew/opt/lapack/lib`
```

Comment:
Let's make this as platform agnostic as possible:
```suggestion
  * macOS: `gem install numo-narray`, `gem install numo-linalg -- --with-openblas-dir=$(brew --prefix openblas) --with-lapack-lib="$(brew --prefix lapack)/lib"
```

(/opt/homebrew is for homebrew for Apple Silicon. homebrew for intel macs is still in /usr/local)